frontend/src/app/components/containers/complaints/outcomes/oucome-by-animal/create-outcome.tsx

```diff
@@ -320,7 +320,7 @@ export const CreateAnimalOutcome: FC<props> = ({
       }
     });
 
-    if (authorizationRef.current && !authorizationRef.current.isValid()) {
+    if (data.drugAuthorization && authorizationRef.current && !authorizationRef.current.isValid()) {
```

Comment:
should this also go in the edit-outcome.tsx file as well?

Reply:
Yes it should, good catch!